### ISSUE TITLE
bug(duplicates): false positives, if exists comments for Stylus selectors

### ISSUE THREAD

**Issue by Kristinita** [rossPatton/stylint#457]
### 1. Summary

**If** I use comments for Stylus selectors, I get:

```text
duplicates error duplicate property or selector, consider merging
```

#### 2.1. Environment

+ Windows 10 Enterprise LTSB 64-bit EN
+ Node.js 12.1.0
+ Stylint 2.0.0

### 3. Files

+ `Kira--Stylint-comments.styl`:

```stylus
p
    // Comment
    // Another comment
    font-size 1.9375rem
    font-style italic

div
    // Kira
    // Goddess!
    font-size 1.3rem
    font-style italic

```

I couldn't find, that such comments are prohibited in Stylus.

+ `.stylintrc`:

```json
{
    "colons": {
        "expect": "never",
        "error": true
    },
    "duplicates": {
        "expect": "always",
        "error": true
    }
}

```

### 4. Steps to reproduce

```text
stylint Kira--Stylint-comments.styl
```

### 5. Actual behavior

**If** comments in `Kira--Stylint-comments.styl` exists:

```js
D:\Kira--Stylint-comments.styl
10 duplicates error duplicate property or selector, consider merging

D:\Kira--Stylint-comments.styl
11 duplicates error duplicate property or selector, consider merging

Stylint: 2 Errors.
Stylint: 0 Warnings.
```

### 6. Expected behavior

**Else** I strip comments:

```stylus
p
    font-size 1.9375rem
    font-style italic

div
    font-size 1.3rem
    font-style italic

```

No warnings or errors.

Thanks.

**Comment by amitYadavDev** [comment]
Hi @Kristinita,  i m going to work on this issue.
